doc/classes/Array.xml

```diff
@@ -383,8 +383,7 @@
 			<param index="0" name="position" type="int" />
 			<param index="1" name="value" type="Variant" />
 			<description>
-				Inserts a new element at a given position in the array. The position must be valid, or at the end of the array ([code]pos == size()[/code]).
-				[b]Note:[/b] This method acts in-place and doesn't return a value.
```

Comment:
You should also keep the note that the array is modified in place. You can adjust "and doesn't return a value" to say "and doesn't return a new array", or something similar to that.